pkg/controller/cincinnati/new.go

```diff
@@ -333,36 +321,105 @@ func (k *kubeResources) newDeployment(instance *cv1beta1.Cincinnati) *appsv1.Dep
 	return dep
 }
 
-func (k *kubeResources) addExternalCACert(instance *cv1beta1.Cincinnati) {
+func (k *kubeResources) newVolumes(instance *cv1beta1.Cincinnati) *[]corev1.Volume {
+	mode := int32(420) // 0644
+	return &[]corev1.Volume{
+		corev1.Volume{
+			Name: "configs",
+			VolumeSource: corev1.VolumeSource{
+				ConfigMap: &corev1.ConfigMapVolumeSource{
+					DefaultMode: &mode,
+					LocalObjectReference: corev1.LocalObjectReference{
+						Name: nameConfig(instance),
+					},
+				},
+			},
+		},
+		corev1.Volume{
+			Name: "cincinnati-graph-data",
+			VolumeSource: corev1.VolumeSource{
+				EmptyDir: &corev1.EmptyDirVolumeSource{},
+			},
+		},
+	}
+}
+
+func (k *kubeResources) newTrustedCAConfig(instance *cv1beta1.Cincinnati, cm *corev1.ConfigMap) {
```

Comment:
By convention, a function that starts with `new` is a constructor that returns something new. Since this function is intended to have a side effect of modifying something that exists vs. returning something new, I suggest a name change to `addTrustedCAConfig` or something similar.

Reply:
Ah, I see you already have an `addTrustedCAConfig` function, and it's the only place where this gets called. What do you think about just moving this code into that function? I'm not seeing the value in having a separate function just to instantiate and assign a ConfigMap, and that would solve the naming confusion.

Reply:
That's fine with me.